src/packages/rlp/package.json

```diff
@@ -49,7 +49,7 @@
   ],
   "dependencies": {
     "@ganache/utils": "0.5.0",
-    "rlp": "2.2.6"
+    "rlp": "2.2.7"
```

Comment:
`@ethereumjs/*` packages have updated to this version of `rlp`, so I matched our dependency with theirs so we don't have to install multiple.